examples/wallet/app/components/RestoreWalletFromPrivateKey.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import typeof { setAccount as SetAccount } from '../actions/account';
+
+type Props = {
+  setAccount: SetAccount
+};
+
+// FIXME: this has no error handling, neither while parsing the address
+// nor when fetching the balance.
+export default ({ setAccount }: Props) => {
+  const handleSubmitCreateSpending = function handleSubmitCreateSpending(
+    event
+  ) {
+    event.preventDefault();
+    if (mustCreateSpendingPassword) {
+      if (checkValidSpendingPassword(password, confirmPassword)) {
+        setAccount(newPrivateKey, password);
```

Comment:
If the created password is invalid the user can submit the form but nothing is done?

I'd rather prevent them from submitting altogether, and at least log an error in this case

Reply:
It was removed